atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/HttpBenchmarks.java

```diff
@@ -51,6 +52,7 @@
                     .build();
 
     @Benchmark
+    @Threads(1)
```

Comment:
Hmm. It seems like we do actually need this - the default if unspecified is the number of processors, and if we specify it at config level it will override a benchmark specific setting.